modules/ztp-high-security-issues-addressed.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-security-hardening.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-addressed-security-issues_{context}"]
+= Security hardening measures implemented
+
+== Disabling USB boot, wireless and bluetooth access  
```

Comment:
```suggestion
== Disabling USB boot and wireless and bluetooth access  
```

Reply:
Or "Disabling USB boot, wireless, and bluetooth access" but the first suggestion seems more accurate depending on whether or not "wireless and bluetooth access" is a thing (together) or not.